### PR TITLE
[Outlook] (error) Document error 9043 AttachmentTypeNotSupported

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Fixes https://github.com/OfficeDev/office-js-docs-pr/issues/1492.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[729dfac](https://github.com/OfficeDev/office-js-docs-pr/commits/729dfac600f007edd564c024d5720463f3863191)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-error-9043/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001301628252812-1574%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3TPBqnSXTexV2g2nPXIA7EDxXoJDYe65KFqhGeAoEPg%3D&st=2020-01-30T16%3A24%3A31Z&se=2020-03-01T16%3A29%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b099910](https://github.com/OfficeDev/office-js-docs-pr/commits/b0999106d0851b2910894530b954bf2e28a36325)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-error-9043/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001301853530268-1574%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ksG7lRA5MNWoxIctLrOq5RdIoaAqs4G%2BCwbVAuYUp8U%3D&st=2020-01-30T18%3A50%3A03Z&se=2020-03-01T18%3A55%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4bcdea0](https://github.com/OfficeDev/office-js-docs-pr/commits/4bcdea0eed10bc481280316692af7760184a1c1a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-error-9043/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001302133499098-1574%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gT5t6is%2BMi%2FqI6Rz68Ht5qZayQJb2tKJsIA3Sa3Gt7k%3D&st=2020-01-30T21%3A29%3A57Z&se=2020-03-01T21%3A34%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9470e2b](https://github.com/OfficeDev/office-js-docs-pr/commits/9470e2b06956628255c917e1ff8263226ef1a26c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-error-9043/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001302208289551-1574%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=9tTu6W9AIQU7qCS9M6qgkqpJuaW7KUiB8sx0HNrmWGQ%3D&st=2020-01-30T22%3A04%3A36Z&se=2020-03-01T22%3A09%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/javascript-api-for-office-error-codes.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ The following table lists the error codes, names, and messages displayed, and th
 |9021|Save Error|Connection error occurred while trying to save the item on the server.|The item couldn't be saved. In Online Mode, this could be due to a server connection error.|
 |9022|Message In Different Store Error|The EWS ID cannot be retrieved because the message is saved in another store.|The EWS ID for the current message couldn't be retrieved as the message may have been moved or the sending mailbox may have changed.|
 |9041|Network error|The user is no longer connected to the network. Please check your network connection and try again.|The user no longer has network or internet access.|
+|9043|Attachment Type Not Supported|The attachment type is not supported.|The API doesn't support the attachment type. For example, `item.getAttachmentContentAsync` throws this error if the attachment is an embedded image in Rich Text Format, or if it's an item type other than an email or calendar item (such as a contact or task item).|
```
Comment (emtencza) [inline docs/reference/javascript-api-for-office-error-codes.md]: "or if it has an item type"

**Pull request merged**